hedera-node/test-clients/src/main/resource/log4j2.xml

```diff
@@ -25,6 +25,17 @@
 			<SizeBasedTriggeringPolicy size="50 MB" />
 			<DefaultRolloverStrategy max="10"/>
 		</RollingFile>
+
+		<!-- State logs -->
+		<RollingFile name="StateLogs"
+								 fileName="output/state/state-changes.log"
+								 filePattern="output/state/state-changes-%d{yyyy-MM-dd--HH-mm-ss}-%i.log.gz">
```

Comment:
We recently changed to have only file rolling without compression. Any reason why this is needed here?

Reply:
No particular reason. I was following `TransactionStateLogs` pattern

Reply:
Should I update `TransactionStateLogs` as well?

Reply:
I fixed both.